mmapi.pm

```diff
@@ -92,6 +92,8 @@ sub api_call {
     return ($res, $tx);
 }
 
+sub api_call { api_call_2(@_)->res }
```

Comment:
please add a method description here and above.

Reply:
Ah, nice, I was just about to ask what the docs now state 👍